packages/ckeditor5-image/ckeditor5-metadata.json

```diff
@@ -7,7 +7,70 @@
 			"docs": "features/image.html#inserting-images-via-pasting-url-into-editor",
 			"path": "src/autoimage.js",
 			"requires": [
-				"Image"
+				[
+					"ImageBlock",
+					"ImageInline"
+				]
+			]
+		},
+		{
+			"name": "Block image",
+			"className": "ImageBlock",
+			"description": "Enables support for inserting images between the blocks of text, like paragraphs or block quotes. This functionality is also provided by the main Image plugin. Note: The image feature is introduced in a granular form, implemented by separate plugins responsible for such aspects as image caption, toolbar, resize, upload or styles.",
+			"path": "src/imageblock.js",
```

Comment:
Missing "docs" property, so the `ImageBlock` plugin will not have a "Documentation" link on the bottom of the tile in Online Builder:

<img src="https://user-images.githubusercontent.com/56868128/123605050-ba762480-d7fb-11eb-8532-8cad23bcf339.png" width="300px">